index.mk

```diff
@@ -19,6 +18,12 @@ endif
 NPM_BIN = ./node_modules/.bin
 export PATH := $(PATH):$(NPM_BIN)
 
+ifeq ("$(wildcard node_modules/@financial-times/origami-service-makefile/index.mk)","")
+PATH_TO_SERVICE_MAKEFILE :=
```

Comment:
What would happen if this condition executed?

Reply:
The env is set but empty. For our purposes though I don't think this is necessary seen as we don't check if it is set.

Reply:
So I'll update this:
```
ifneq ("$(wildcard node_modules/@financial-times/origami-service-makefile/index.mk)","")
PATH_TO_SERVICE_MAKEFILE := node_modules/@financial-times/origami-service-makefile/
endif
```